src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -530,4 +616,16 @@ const StyledLabel = styled.div`
   transition: all 0.2s ease-in-out;
   border-radius: 4px !important;
 `
+
+const Radio = styled.input`
```

Comment:
please use Bootstrap for this

Reply:
AND ALSO CAN'T UNDERSTAND THIS ONE HOW USE BOOTSTRAP BECAUSE WE ARE USING STYLED-COMPONENTS?

Reply:
https://react-bootstrap.netlify.app/docs/forms/checks-radios/